moped-etl/prefect/moped_test_create_flow.py

```diff
@@ -267,12 +269,12 @@ def drain_service(slug):
 
     ## Commission the Activity Log
 
-    commission_activity_log_command = activity_log.create_activity_log_command(
-        slug=slug, upstream_tasks=[git_repo_checked_out]
-    )
-    deploy_activity_log = activity_log.create_activity_log_task(
-        command=commission_activity_log_command
-    )
+    # commission_activity_log_command = activity_log.create_activity_log_command(
+    #     slug=slug, upstream_tasks=[git_repo_checked_out]
+    # )
+    # deploy_activity_log = activity_log.create_activity_log_task(
+    #     command=commission_activity_log_command
+    # )
```

Comment:
I commented these out since I dont think the activity log in test deployments work anyway

Reply:
Thanks to your work on this we won't have to worry about these resources at all soon!